_includes/auth/private-public-key-pairs-for-tiny-cloud-services.md

```diff
@@ -1,9 +1,21 @@
-## Private/public key pairs for {{site.cloudname}} services
+## Add a public key to the {{site.cloudname}} API key
 
-{{site.cloudname}} services tokens use public/private RSA key pairs. The {{site.cloudname}} services only store the public key, allowing developers to have full control over the authentication.
+The {{pluginname}} Server requires a _public_ key generated from the same _private_ key that will be used on your JSON Web Token (JWT) Provider server. The public key(s) stored on the {{pluginname}} Server are used to ensure that content is sent by authorized users.
```

Comment:
This uses different terms to what's used elsewhere which could cause some confusion? e.g here it uses `JSON Web Token (JWT) Provider server` but in how-jwts-are-used.md it uses `JSON Web Token (JWT) endpoint`

Reply:
We also use `endpoint` in jwt-authentication.md, so whatever we change it to we'll need to make sure that's updated as well.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/2091/commits/b3091a1ad3845121811a20d729af9663174b26ec